core/server/data/schema/schema.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
         slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
         secret: {type: 'string', maxlength: 150, nullable: false},
         redirection_uri: {type: 'string', maxlength: 2000, nullable: true},
+        blog_uri: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Ok understand. Good idea.
I looked at some references.

https://developers.google.com/identity/protocols/OAuth2UserAgent
https://developer.github.com/v3/oauth/
https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow

Nobody uses or defines more then the default oauth2 spec parameters (redirect_uri, scope, client_id, client_secret, state). 

Then i looked into the RFC spec again
https://tools.ietf.org/html/rfc7591

This is an example request to register a client (https://tools.ietf.org/html/rfc7591#page-16)
```
     POST /register HTTP/1.1
     Content-Type: application/json
     Accept: application/json
     Host: server.example.com

     {
      "redirect_uris": [
        "https://client.example.org/callback",
        "https://client.example.org/callback2"],
      "client_name": "My Example Client",
      "client_name#ja-Jpan-JP":
         "\u30AF\u30E9\u30A4\u30A2\u30F3\u30C8\u540D",
      "token_endpoint_auth_method": "client_secret_basic",
      "logo_uri": "https://client.example.org/logo.png",
      "jwks_uri": "https://client.example.org/my_public_keys.jwks",
      "example_extension_parameter": "example_value"
     }
```
^ BTW: they suggest using `client_name` and not `name`.

And they have listed more attributes here https://tools.ietf.org/html/rfc7591#page-9

```
   client_uri
      URL string of a web page providing information about the client.
      If present, the server SHOULD display this URL to the end-user in
      a clickable fashion.  It is RECOMMENDED that clients always send
      this field.  The value of this field MUST point to a valid web
      page.  The value of this field MAY be internationalized, as
      described in Section 2.2.
```

So i would change it to `client_uri` if you agree 👍